config.js

```diff
@@ -82,6 +82,41 @@ config.oauth2 = {
     }
 }*/
 
+config.oauth2 = {
```

Comment:
That is not needed anymore? It's overwriting the definition above

Reply:
Yes, forgot to remove it. Fixed it